templates/miq-template.yaml

```diff
@@ -442,7 +585,7 @@ parameters:
     displayName: "Application Min CPU Requested"
     required: true
     description: "Minimum amount of CPU time the Application container will need (expressed in millicores)."
-    value: "1000m"
+    value: "900m"
```

Comment:
Getting a new memory usage footprint when idling should be straight forward using a Openshift cluster with metrics, we can adjust as we have hard data on it.